docs/docs-content/enterprise-version/upgrade/upgrade-vmware/airgap.md

```diff
@@ -0,0 +1,186 @@
+---
+sidebar_label: "Airgap"
+title: "Upgrade Airgap Palette Installed on VMware vSphere"
+description: "Learn how to upgrade self-hosted airgap Palette in VMware."
+icon: ""
+sidebar_position: 10
+tags: ["palette", "self-hosted", "vmware", "airgap", "upgrade"]
+keywords: ["self-hosted", "enterprise"]
+---
+
+This guide takes you through the process of upgrading a self-hosted airgap Palette instance installed on VMware vSphere.
+
+:::warning
+
+Before upgrading Palette to a new major version, you must first update it to the latest minor version available. Refer
+to the [Supported Upgrade Paths](../upgrade.md#supported-upgrade-paths) section for details.
+
+If your setup includes a PCG, you must also
+[allow the PCG to upgrade automatically](../../../clusters/pcg/manage-pcg/pcg-upgrade.md) before each major or minor
+Palette upgrade. Additionally, if the new Palette version updates Kubernetes, you need to apply the Kubernetes updates
+to your Enterprise Cluster profile.
+
+:::
+
+## Prerequisites
+
+- Access to the Palette system console.
+
+- Access to the Palette airgap support Virtual Machine (VM) that you used for the initial Palette installation.
+
+- Refer to [Access Palette](/enterprise-version/#access-palette) to download the following from our support:
+  - The new airgap Palette installation bin.
+  - The OVA with the Operating System (OS) and Kubernetes distribution required for the Palette nodes (if the new
+    version of Palette requires a new version of the OS and Kubernetes).
+
+## Upgrade
+
+1.  Use the following command template to SSH into the Palette airgap support VM. Enter the path to your private SSH
+    key, your username, and the IP or domain of the airgap support VM. The default username is `ubuntu`.
+
+    ```shell
+    ssh -identity_file </path/to/private/key> <username>@<vm-ip-or-domain>
```

Comment:
the username for the airgap support VM instance is `ubuntu` so I think we can replace `<username>` with the default user we create.